### DISCUSSION TITLE
blog/respond-peer-reviews-with-pandoc/

### DISCUSSION THREAD

**Discussion by giscus** [gadenbuie/garrickadenbuie-com#132] [Comments]
# blog/respond-peer-reviews-with-pandoc/

Data Scientist. #rstats. Educator. Musician. Always learning something new.

https://www.garrickadenbuie.com/blog/respond-peer-reviews-with-pandoc/

**Comment by zxdawn** [comment]
Thanks for sharing! Do you have any example of the `response.txt` file?
